src/Classes/Utils/CanvasUtils.ts

```diff
@@ -0,0 +1,348 @@
+import {IDimension} from "../../Interfaces/IDimension";
+import {ICoordinates} from "../../Interfaces/ICoordinates";
+import {IRgb} from "../../Interfaces/IRgb";
+import {Particle} from "../Particle";
+import {Utils} from "./Utils";
+import {IStroke} from "../../Interfaces/Options/Particles/Shape/IStroke";
+import {ShapeType} from "../../Enums/ShapeType";
+import {ISide} from "../../Interfaces/ISide";
+
+export class CanvasUtils {
+    public static paintBase(context: CanvasRenderingContext2D, dimension: IDimension, baseColor: string = "rgba(255, 255, 255, 0)"): void {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5171185)